src/server/AppManager.ts

```diff
@@ -202,7 +202,7 @@ export class AppManager {
                 continue;
             }
 
-            await this.initializeApp(items.get(rl.getID()), rl, true).catch(console.error);
+            await this.initializeApp(items.get(rl.getID()), rl, false, true).catch(console.error);
```

Comment:
Hi! Can you explain the details? Why change `saveToDb = true` to `false` can help solve the issue?

Reply:
Yes! I updated the PR description

Reply:
Nice! Thanks for th detailed explanation!